.azure/pipelines/stress.yml

```diff
@@ -0,0 +1,38 @@
+# This configuration builds the repository and runs stress
+
+# Don't run CI for this config
+trigger:
+  batch: true
+  branches:
+    include:
+    - 'master'
+
+variables:
+- name: DOTNET_SKIP_FIRST_TIME_EXPERIENCE
+  value: true
+- name: _TeamName
+  value:  AspNetCore
+- ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
+  - name: _BuildArgs
+    value: /p:TeamName=$(_TeamName)
+      /p:OfficialBuildId=$(Build.BuildNumber)
+- ${{ if or(eq(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'PullRequest')) }}:
+  - name: _BuildArgs
+    value: ''
+jobs:
+- template: jobs/default-build.yml
+  parameters:
+    condition: ne(variables['SkipTests'], 'true')
+    jobName: Windows_Stress_Test
+    jobDisplayName: "Windows Stress test"
+    agentOs: Windows
+    isTestingJob: true
+    steps:
+    - script: .\src\Servers\Kestrel\stress\build.cmd -ci
```

Comment:
I like how relatively simple this is. I'm going to try and run it locally to see how reproducible it is :).

Reply:
Sorry, what are you trying to reproduce?